modules/distribution/resources/operation_policies/specifications/opaPolicy.json

```diff
@@ -0,0 +1,94 @@
+{
+  "category": "Mediation",
+  "name": "opaPolicy",
+  "displayName": "Validate Request With OPA Policy",
+  "description": "With this policy, user can validate requests based on the OPA policy engine",
+  "multipleAllowed": true,
+  "policyAttributes": [
+    {
+      "name": "serverUrl",
+      "displayName": "OPA server URL",
+      "description": "OPA server's URL",
+      "validationRegex": "([\\w+]+\\:\/\/)?([\\w\\d-]+\\.)*[\\w-]+[\\.\\:]\\w+([\/\\?\\=\\&\\#\\.]?[\\w-]+)*\/?",
```

Comment:
This regex do not support the URL `https://helloworld`, it required dot or colon in the middle

Reply:
btw this can be added as `https://helloworld:443`

Reply:
fixed